test/DynamoCoreTests/SerializationTests.cs

```diff
@@ -180,7 +177,8 @@ private void CompareWorkspaces(WorkspaceComparisonData a, WorkspaceComparisonDat
             }
             Assert.AreEqual(a.NodeCount, b.NodeCount, "The workspaces don't have the same number of nodes.");
             Assert.AreEqual(a.ConnectorCount, b.ConnectorCount, "The workspaces don't have the same number of connectors.");
-            Assert.AreEqual(a.GroupCount, b.GroupCount, "The workspaces don't have the same number of groups.");
+            //TODO: Annotations tests should be in viewmodel serialization tests.
+           // Assert.AreEqual(a.GroupCount, b.GroupCount, "The workspaces don't have the same number of groups.");
```

Comment:
just left a TODO here.  I think these tests should be added to ViewModel SerializationTests.